sample/Broadcast/broadcast_sample.rb

```diff
@@ -0,0 +1,96 @@
+require 'sinatra/base'
+require 'opentok'
+
+raise "You must define API_KEY and API_SECRET environment variables" unless ENV.has_key?("API_KEY") && ENV.has_key?("API_SECRET")
+
+class BroadcastSample < Sinatra::Base
+
+  set :api_key, ENV['API_KEY']
+  set :opentok, OpenTok::OpenTok.new(api_key, ENV['API_SECRET'])
+  set :session, opentok.create_session(:media_mode => :routed)
+  set :erb, :layout => :layout
+  set :broadcastId, nil
+  set :focusStreamId, ''
+  set :broadcastLayout, 'bestFit'
+
+  get '/' do
+    erb :index
+  end
+
+  get '/host' do
+    api_key = settings.api_key
+    session_id = settings.session.session_id
+    token = settings.opentok.generate_token(session_id, role: :publisher, initialLayoutClassList: ['focus'])
+
+    erb :host, locals: {
+        apiKey: api_key,
+        sessionId: session_id,
+        token: token,
+        initialBroadcastId: settings.broadcastId,
```

Comment:
Please use `snake_case` for property names in Ruby (to be consistent with the existing names and the [Ruby style guide](https://github.com/rubocop-hq/ruby-style-guide#naming)). So use: `broadcast_id`, `focus_stream_id`, and `broadcast_layout`. (But use`camelCase` for the Javascript property names.)